server/util-server.js

```diff
@@ -203,6 +204,29 @@ exports.dnsResolve = function (hostname, resolverServer, rrtype) {
     });
 };
 
+/**
+ * Run a query on SQL Server
+ * @param {string} connectionString The database connection string
+ * @param {string} query The query to validate the database with
+ * @returns {Promise<(string[]|Object[]|Object)>}
+ */
+exports.sqlserver = function (connectionString, query) {
```

Comment:
Should have a better function name.

Reply:
What is your preference? sqlserverDatabase?

Reply:
I changed it to `mssqlQuery` to match the import name + what it is actually doing.